Gemfile.lock

```diff
@@ -1,11 +1,80 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.5.0)
+    activesupport (4.2.7)
+      i18n (~> 0.7)
+      json (~> 1.7, >= 1.7.7)
+      minitest (~> 5.1)
+      thread_safe (~> 0.3, >= 0.3.4)
+      tzinfo (~> 1.1)
+    addressable (2.5.1)
       public_suffix (~> 2.0, >= 2.0.2)
+    coffee-script (2.4.1)
+      coffee-script-source
+      execjs
+    coffee-script-source (1.12.2)
     colorator (1.1.0)
-    ffi (1.9.14)
+    ethon (0.10.1)
+      ffi (>= 1.3.0)
+    execjs (2.7.0)
+    faraday (0.12.0.1)
+      multipart-post (>= 1.2, < 3)
+    ffi (1.9.18)
     forwardable-extended (2.6.0)
+    gemoji (3.0.0)
+    github-pages (126)
+      activesupport (= 4.2.7)
+      github-pages-health-check (= 1.3.3)
+      jekyll (= 3.3.1)
+      jekyll-avatar (= 0.4.2)
+      jekyll-coffeescript (= 1.0.1)
+      jekyll-default-layout (= 0.1.4)
+      jekyll-feed (= 0.9.1)
+      jekyll-gist (= 1.4.0)
+      jekyll-github-metadata (= 2.3.1)
+      jekyll-mentions (= 1.2.0)
+      jekyll-optional-front-matter (= 0.1.2)
+      jekyll-paginate (= 1.1.0)
+      jekyll-readme-index (= 0.0.4)
+      jekyll-redirect-from (= 0.12.1)
+      jekyll-relative-links (= 0.3.0)
+      jekyll-sass-converter (= 1.5.0)
+      jekyll-seo-tag (= 2.1.0)
+      jekyll-sitemap (= 1.0.0)
+      jekyll-swiss (= 0.4.0)
+      jekyll-theme-architect (= 0.0.3)
```

Comment:
Yeah I didn't look through it yet, but it did feel like it added what may just be a ton of unnecessary cruft. Lemme see what we actually need / what we don't. 👍